docs/explanations/how_to_interpret_the_security_status_command.md

```diff
@@ -1,7 +1,186 @@
 # What does `security-status` do?
 
+## Command output
+
 The `security-status` command is used to get an overview of the packages
-installed on your machine.
+installed on your machine, and an overview of the security coverage for those
+packages.
```

Comment:
Can't use suggestion because of the deleted line, but I think this could be changed to the following (or words to that effect):

> The `security-status` command provides an overview of all the packages installed on your machine, and the security coverage that applies to those packages.